test/pkg/environment/aws/environment.go

```diff
@@ -35,28 +36,36 @@ type Environment struct {
 	*common.Environment
 	Region string
 
-	EC2API ec2.EC2
-	SSMAPI ssm.SSM
-	STSAPI sts.STS
-	IAMAPI iam.IAM
+	EC2API         ec2.EC2
+	SSMAPI         ssm.SSM
+	STSAPI         sts.STS
+	IAMAPI         iam.IAM
+	SQSAPI         *sqs.SQS
+	EventBridgeAPI *eventbridge.EventBridge
 
-	SQSProvider     *providers.SQS
-	InterruptionAPI *itn.ITN
+	SQSProvider         *providers.SQS
+	EventBridgeProvider *providers.EventBridge
+	InterruptionAPI     *itn.ITN
 }
 
 func NewEnvironment(t *testing.T) *Environment {
 	env := common.NewEnvironment(t)
 	session := session.Must(session.NewSessionWithOptions(session.Options{SharedConfigState: session.SharedConfigEnable}))
 
-	return &Environment{
+	ret := &Environment{
 		Region:          *session.Config.Region,
 		Environment:     env,
 		EC2API:          *ec2.New(session),
 		SSMAPI:          *ssm.New(session),
 		IAMAPI:          *iam.New(session),
+		SQSAPI:          sqs.New(session),
+		EventBridgeAPI:  eventbridge.New(session),
```

Comment:
nit: is there a reason these are pointers? I don't think it really matters which way we go, but I think they should all be values or all pointers :D

Reply:
They should probably all be pointers since that's what the New function produces